python/nav/models/sql/changes/sc.05.00.0080.sql

```diff
@@ -0,0 +1,14 @@
+BEGIN
```

Comment:
2 problems with this:

1. There is a syntax error (missing semicolon) causing the CI build to fail.
2. There is no need for `BEGIN/END` statements. `navsyncdb` runs each changescript in a separate transaction.